content/hi/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,129 @@
+---
+# reviewers:
+# - lavalamp
+title: कुबेरनेट्स घटक
+content_type: concept
+description: >
+  कुबेरनेट्स क्लस्टर में कंट्रोल प्लेन के घटक और मशीनों का एक समूह होता है।
+  विमान और मशीनों का एक सेट होता है जिसे नोड्स कहा जाता है।
+weight: 30
+card:
+  name: concepts
+  weight: 20
+---
+
+<!-- overview -->
+जब आप कुबेरनेट्स को डिप्लॉय करते हैं, तो आपको एक क्लस्टर मिलता है।
+{{< glossary_definition term_id="cluster" length="all" prepend="कुबेरनेट्स क्लस्टर में">}}
+
+यह दस्तावेज़ आपके लिए आवश्यक विभिन्न घटकों की रूपरेखा देता है
+एक पूर्ण और कार्यशील कुबेरनेट क्लस्टर।
+
+{{< figure src="/images/docs/components-of-kubernetes.svg" alt="Components of Kubernetes" caption="The components of a Kubernetes cluster" class="diagram-large" >}}
+
+<!-- body -->
+## कंट्रोल प्लेन घटक
+
+कंट्रोल प्लेन के घटक क्लस्टर (उदाहरण के लिए, शेड्यूलिंग) के बारे में समग्र निर्णय लेते हैं, साथ ही साथ क्लस्टर घटनाओं का पता लगाते हैं और उनका जवाब देते हैं (उदाहरण के लिए, एक नया {{< glossary_tooltip text="पॉड" term_id="pod">}} शुरू करना जब डिप्लॉयमेंट का `replicas` फ़ील्ड असंतुष्ट हो)।
+
+कंट्रोल प्लेन घटकों को क्लस्टर में किसी भी मशीन पर चलाया जा सकता हैं। हालांकि,
+आसानी के लिए, सेट अप स्क्रिप्ट आमतौर पर पर एक ही मशीन पर सभी कंट्रोल प्लेन
```

Comment:
आसानी के लिए, सेट अप स्क्रिप्ट आमतौर पर  एक ही मशीन पर सभी कंट्रोल प्लेन

This sentence will be good as there is twice repetation of same word in it.